aquadoggo/migrations/20220510022755_create-documents.sql

```diff
@@ -0,0 +1,24 @@
+-- SPDX-License-Identifier: AGPL-3.0-or-later
+
+CREATE TABLE IF NOT EXISTS document_view_fields (
```

Comment:
It somewhere makes sense to _not_ use version numbers in these tables but I also don't see a reason to _not_ do it, what do you think?

Reply:
Yeh, it's an interesting question. I suppose the document view doesn't really know what version it's been materialised from, so that's the reason we could omit the version. Are there cases where it would need to know?... I'm not sure...

Reply:
"Document" and "Document View" are defined in the p2panda specification and then also somewhat concerning a first "specification version". We might change something with them as well in the future, who knows. On the other hand, they represent materialized data which can be easily deleted and re-built after a specification change. Maybe thats reason enougn to _not_ give them a version in the tables :+1:

Reply:
> they represent materialized data

This kind of was my thinking why we don't need versioned tables. Document views aren't part of the transported, long term stored, p2panda data layer, they are more ephemeral. 

But I definitely don't have the mental foresight to understand the implications of not having versions :rofl:. Operations are versioned, so then that is clear we need it, but does that trickle up to document views?? Oh god, I have no idea.

Reply:
I think it's alright to leave it out now, it's always possible to change table names as well 🤣